omnibus/omnibus.rb

```diff
@@ -35,7 +35,7 @@
 
 # Enable S3 asset caching
 # ------------------------------
-use_s3_caching true
+use_s3_caching false 
```

Comment:
If you plan on sending a build through CI you need to leave that setting as is or bad things will happen...trust me from experience here.